sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_do_while.py

```diff
@@ -38,20 +38,19 @@ def pipeline_with_do_while_node():
 
     :param body: Pipeline job for the do-while loop body.
     :type body: Pipeline
-    :param condition: Name of boolean output of body as do-while loop condition.
-    :type condition: Output
     :param mapping: Output-Input mapping for reach round of the do-while loop.
                     Key is the last round output of the body. Value is the input port for current body.
     :type mapping: Dict[Union[str, Output], Union[str, Input, List]]
     :param max_iteration_count: limits in running the do-while node.
     :type max_iteration_count: int
+    :param condition: Name of boolean output of body as do-while loop condition; handle as while true when omitted.
```

Comment:
Name of a boolean output of body, do-while loop stops if it's value is evaluated to be false; If not specified, handle as while true.

Reply:
Updated as this comment suggests.